book/listings/gobject_signals/3/custom_button/mod.rs

```diff
@@ -1,5 +1,6 @@
 mod imp;
 
+use glib::Object;
```

Comment:
not sure this is worth it, you only use it once

Reply:
Good point.
I'd still prefer to keep it because I tend to use it in long lines